tests/gdimagefill/bug00104_1.c

```diff
@@ -0,0 +1,40 @@
+/* $Id$ */
```

Comment:
drop this $Id$ ... it isn't useful anymore.  just put a description of the bug you're testing for here.